src/store/metadata/oxigraph/store.py

```diff
@@ -110,7 +117,73 @@ def get_edition(
         """
         Gets a specific edition of a specific dataset
         """
-        raise NotImplementedError
+        # Populate the graph from the database
+        graph = self.db
+
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a single Graph of the
+        # data we need.
+        result: Graph = (
+            construct_edition_core(graph, dataset_id, edition_id)
+            + construct_edition_contact_point(graph, dataset_id, edition_id)
+            + construct_edition_themes(graph, dataset_id, edition_id)
+            + construct_edition_keywords(graph, dataset_id, edition_id)
+            + construct_edition_temporal_coverage(graph, dataset_id, edition_id)
+            + construct_edition_table_schema(graph, dataset_id, edition_id)
+            + construct_edition_versions(graph, dataset_id, edition_id)
+        )
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "dcat:Dataset"}
+        )
+
+        edition_graph = next((x for x in data["@graph"] if "@type" in x.keys()), None)
+        contact_point_graph = next(
+            (x for x in data["@graph"] if "vcard:fn" in x.keys()), None
+        )
+        temporal_coverage_graph = next(
+            (x for x in data["@graph"] if "dcat:endDate" in x.keys()), None
+        )
+        table_schema_graph = next(
+            (x for x in data["@graph"] if "columns" in x.keys()), None
+        )
+        columns_graph = [x for x in data["@graph"] if "datatype" in x.keys()]
+
+        versions_graph = next((x for x in data["@graph"]))
+        # Populate table_schema_graph.columns with column definitions and delete table_schema_graph.column blank node @id
+        for column in table_schema_graph["columns"]:
```

Comment:
if I'm following you should be able to do something like:

```
for column in columns_graph:
      del column["@id"]
```

and just `edition_graph["table_schema"]["columns"] = columns_graph` later on